great_expectations/core/http.py

```diff
@@ -19,6 +19,8 @@ def _log_request_method_and_response(r: requests.Response, *args, **kwargs):
         LOGGER.debug(f"{r}\n{pf(r.json(), depth=3)}")
     except json.JSONDecodeError:
         LOGGER.debug(f"{r}\n{r.content.decode()}")
+    except Exception as other_err:
+        LOGGER.info(f"{r} - Error logging response {other_err!r}")
```

Comment:
We were getting errors when the agent is run on Databricks (likely a dependency issue).
Likely this will just move the error to when we attempt to parse the json response but it's still an improvement.

Reply:
Nice, also - TIL about `!r`.

Reply:
I'm doing `!r` because I want to see the `__repr__` of the exception instead of just the `__str__` (`!s`).
The `__repr__` includes the name of the exception whereas the `__str__` only shows the exception message.